### PR TITLE
[selinux] initial integration

### PR DESCRIPTION
**Pull request by evverx**

### CODE REVIEW

**Comment by evverx** [comment]
The ASan tests are failing due to a bug in the parser the fuzz target is supposed to test. Hopefully It will go away once https://github.com/SELinuxProject/selinux/pull/274 is merged.

File: projects/selinux/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+FROM gcr.io/oss-fuzz-base/base-builder
+RUN apt-get update &&\
```
Comment (jonathanmetzman) [inline projects/selinux/Dockerfile:18]: nit: space after `&&`  and before `\`.

**Review by jonathanmetzman — APPROVED** [review]
We approve of adding this project. Just retrigger CI when the fix lands.
Things get messed up if we merge a new project whose builds fail immediately.

**Comment by evverx** [comment]
@jonathanmetzman I think the PR should be good to go.

**Comment by jonathanmetzman** [comment]
Nice!

**Comment by fishilico** [comment]
@jonathanmetzman I am receiving crash reports from the fuzzer (which is nice, thanks!) but when I log in to https://oss-fuzz.com/ I get an error message:
```
You (email=nicolas***@gmail.com) are not authorized to access this page!

Please contact the maintainers by filing a new issue on https://github.com/google/oss-fuzz.
```

And on https://bugs.chromium.org/p/oss-fuzz/issues:
```
Permission denied.
```

My email address which is used as primary contact (nicolas.iooss_ossfuzzselinux@m4x.org) is an alias of my GMail account, as documented in "Sign in to your Google Account with another email address", https://support.google.com/accounts/answer/176347?hl=en.

So it seems that the authentication on https://oss-fuzz.com/ is broken: I logged in my Google account in a new "Chrome incognito browsing window", with `nicolas.iooss_ossfuzzselinux@m4x.org` as login, and https://oss-fuzz.com/ still sees my primary GMail account.

 I really want to receive the reports on this "tagged" email address instead of my bare account. How can I do that?

**Comment by evverx** [comment]
> I really want to receive the reports on this "tagged" email address instead of my bare account. How can I do that?

To judge from https://google.github.io/oss-fuzz/faq/#why-do-you-require-a-google-account-for-authentication, it seems alternate email addresses can't be used to access oss-fuzz.com. I'm not sure why https://google.github.io/oss-fuzz/getting-started/new-project-guide/#primary still points to https://support.google.com/accounts/answer/176347?hl=en without mentioning its limitations.

**Comment by jonathanmetzman** [comment]
Sorry for the trouble @fishilico I'll try to update the docs to reflect the limitations.

**Comment by evverx** [comment]
@fishilico I'm not sure how to proceed from here. Should I add your gmail address to https://github.com/google/oss-fuzz/blob/master/projects/selinux/project.yaml?

**Comment by fishilico** [comment]
I created a new Google account to be able to log in, and it seems to work: `https://oss-fuzz.com/testcase-detail/...` now shows:
```
You (email=nicolas.iooss.ossfuzz@gmail.com) are not authorized to access this page!
```

Could you please add `nicolas.iooss.ossfuzz@gmail.com` to the list of account allowed to access the results, while also keeping `nicolas.iooss_ossfuzzselinux@m4x.org` so that I receive the emails? If you can't, I will try to figure out out to configure some email redirection in GMail, if this is possible.

**Comment by evverx** [comment]
Done in https://github.com/google/oss-fuzz/pull/4814

**Pull request merged**